lib/index.ts

```diff
@@ -50,6 +51,34 @@ function getPine(param: {
 	 * This subclass makes use of the [balena-request](https://github.com/balena-io-modules/balena-request) project.
 	 */
 	class BalenaPine extends PinejsClientCore<BalenaPine> {
+		public get(params: Params) {
+			return Promise.resolve(super.get(params));
+		}
+
+		public put(params: Params) {
+			return Promise.resolve(super.put(params));
+		}
+
+		public patch(params: Params) {
+			return Promise.resolve(super.patch(params));
+		}
+
+		public post(params: Params) {
+			return Promise.resolve(super.post(params));
+		}
+
+		public delete(params: Params) {
+			return Promise.resolve(super.delete(params));
+		}
+
+		public upsert(params: UpsertParams) {
+			return Promise.resolve(super.upsert(params));
+		}
+
+		public request(params: Params) {
+			return Promise.resolve(super.request(params));
+		}
```

Comment:
Why are you trying to preserve bluebird? I'd say just drop it completely

Reply:
I was just thinking of a way to make this less major change, in an effort to make it quickly part of the orgs SDK, but we have now already released that.
I also like the convenience methods that bluebird offers, and know that we use them a lot in the SDK and the UI, but if you insist we can drop bluebird as well with this.

Reply:
On the other hand, the SDK already wraps all calls (99% sure)
https://github.com/balena-io/balena-sdk/blob/71a74cebdbe4ff0be728e759b9e4633c46ca39dc/lib/util/callbacks.ts#L18-L33

so it probably is safe to drop this from here.

Reply:
I'd say drop it now and if you want those convenience methods in the SDK it can do
```ts
Bluebird.resolve(pine.get(...)).get('field')
```
```ts
Bluebird.map(pine.get(...), (v) => ...)
```
etc in the places it actually needs it. That way we only have the breaking change in this module once, keep it simpler, and can drop the bluebird dependency here altogether. It also makes dropping bluebird from the SDK simpler in future